go.mod

```diff
@@ -1,3 +1,9 @@
 module github.com/stmcginnis/gofish
 
 go 1.16
+
+require (
```

Comment:
Pulling in dependencies should be avoided. We want to keep this library as light weight as possible, so effort has been made to avoid adding anything.